crates/api_common/src/site.rs

```diff
@@ -108,10 +115,19 @@ pub struct GetSiteResponse {
   pub banned: Vec<PersonViewSafe>,
   pub online: usize,
   pub version: String,
-  pub my_user: Option<LocalUserSettingsView>,
+  pub my_user: Option<MyUserInfo>,
   pub federated_instances: Option<FederatedInstances>, // Federation may be disabled
 }
 
+#[derive(Serialize)]
+pub struct MyUserInfo {
+  pub local_user_view: LocalUserSettingsView,
+  pub follows: Vec<CommunityFollowerView>,
+  pub moderates: Vec<CommunityModeratorView>,
+  pub community_blocks: Vec<CommunityBlockView>,
+  pub person_blocks: Vec<PersonBlockView>,
+}
```

Comment:
The new `MyUserInfo` on `GetSite` contains all your user info. Makes sense to use this, rather than have 5 separate endpoints for logged in user data.